pkg/configuration/configuration_test.go

```diff
@@ -151,5 +153,42 @@ func TestRegistrationService(t *testing.T) {
 		assert.InDelta(t, float32(0.5), regServiceCfg.Verification().CaptchaRequiredScore(), 0.01)
 		assert.False(t, regServiceCfg.Verification().CaptchaAllowLowScoreReactivation())
 		assert.Equal(t, "example-content", regServiceCfg.Verification().CaptchaServiceAccountFileContents())
+		assert.False(t, regServiceCfg.PublicViewerEnabled())
 	})
 }
+
+func TestPublicViewer(t *testing.T) {
```

Comment:
minor - should we name it :
```suggestion
func TestPublicViewerConfiguration(t *testing.T) {
```